src/components/layout/layout.js

```diff
@@ -21,16 +21,18 @@ export default function Layout({ children }) {
   return (
     <Container fluid>
       <Row className={styles.header}>
-        <StaticImage
-          className={styles.headerImg}
-          src="../../images/logo.png"
-          alt="logo"
-        />
-        <Link to="/">
-          <h1>{site.siteMetadata.title}</h1>
-          <h2>Performance Comparison</h2>
+        <Link to="/" className={styles.logoWithText}>
+          <StaticImage
+            src="../../images/logo.png"
```

Comment:
Can we rm the old one? Or replace it with this new one? (right now it's in `images/`). No preference